dlgr/griduniverse/experiment.py

```diff
@@ -1318,11 +1318,24 @@ def game_loop(self):
         self.publish({'type': 'stop'})
         return
 
-    def analyze(self, data):
-        return self.average_score(data)
+    def player_feedback(self, data):
+        df = data.infos.df
+        engagement = int(json.loads(df.questions.list[-1][-1])['engagement'])
```

Comment:
After looking at data.py, I'm not exactly sure how getting the infos table as a list is preferable to getting the infos table as a dataframe.